languages/rusin/pack/src/main/java/com/anysoftkeyboard/languagepack/rusin/PackBroadcastReceiver.java

```diff
@@ -1,7 +1,7 @@
-package com.anysoftkeyboard.languagepack.serbian;
-
-import com.anysoftkeyboard.addon.PackBroadcastReceiverBase;
-
-public class PackBroadcastReceiver extends PackBroadcastReceiverBase {
-
-}
+package com.anysoftkeyboard.languagepack.rusin;
```

Comment:
Did you moved this file from the Serbian pack?

Reply:
Yes, it is based on Serbian keyboard leyout. 
Layout is for Rusyn language for Serbia as country. 
We have same kaybord, the majority of keys are at same position, there are some modifications.

Reply:
The ties to Serbian dictionary are removed. there should not be and tracable connection.